### ISSUE TITLE
Invalid sections in openapi spec

### ISSUE THREAD

**Issue by mikebell** [alphagov/content-store#460]
I'm working with the Tech Docs team to add functionality to their gem.

We're using the Content Store Api yaml file as a test document for our parser integration and have found two issues with it:

1 - https://github.com/alphagov/content-store/blob/master/openapi.yaml#L167 not allowed description here
2 - https://github.com/alphagov/content-store/blob/master/openapi.yaml#L279 - the data should be in a values node

Hopefully that's helpful.

Mike

**Comment by stevenjmesser** [comment]
@MatMoore Has this been dealt with in GTD-2?

**Comment by MatMoore** [comment]
Nope, that issue is just to add support for rendering a spec into the tech docs gem, content store was used for testing it, but we didn't actually change anything here.

This issue is just to say that this spec contains some invalid bits, and I believe https://github.com/alphagov/content-performance-manager/blob/master/openapi.yaml does as well since it was based on content store's one.

AFAIK it's not urgent as it doesn't break anything but it will stop you from migrating away from the current build process that relies on an [alphagov fork of widdershins](https://github.com/alphagov/widdershins) which is not being maintained by anyone and is quite buggy in my experience.

The long term goal is for tech docs sites to use the new built in functionality for API references, although at the moment there are some examples that don't render nicely in that either.
